examples/amazon.temple

```diff
@@ -0,0 +1,64 @@
+Amazon: project {
+  #language(go);
+  #database(postgres);
+  #authMethod(email);
+  #provider(dockerCompose);
+}
+
+Seller: service {
+  firstName: string;
+  lastName: string;
+
+  #auth;
+  #readable(by: all);
+  #enumerable;
+}
+
+Customer: service {
+  firstName: string;
+  lastName: string;
+  addressLine1: string;
+  addressLine2: string;
+  town: string;
+  county: string;
+  postcode: string;
+
+  #auth;
+}
+
+ProductCategory: service {
+  name: string;
+  #readable(by: all);
+  #enumerable;
+}
+
+Product: service {
+  category: ProductCategory;
+  seller: Seller;
+  name: string;
+  description: string;
+  price: float;
+
+  Picture: struct {
+    product: Product;
```

Comment:
Can be omitted because of implicit `parentID`

Reply:
Yes indeed, nice catch